frontend/.env

```diff
@@ -1,3 +1,3 @@
-AUTH_REALM="Demo"
-AUTH_URL="http://localhost:8081/auth/"
-AUTH_CLIENTID="your-client-id"
+AUTH_REALM=Test
```

Comment:
Do we want to set the client id to something that can be used out of the box? Something like `test-client-id`. Just so it doesn't seem like this field will have to be modified before running

Reply:
Sure, I've changed this to reflect `test-client-id`